buildSrc/src/main/groovy/accord.java-conventions.gradle

```diff
@@ -45,3 +45,17 @@ dependencies {
     testImplementation 'ch.qos.logback:logback-classic:1.2.3'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
 }
+
+task copyMainDependencies(type: Copy) {
+    from configurations.compileClasspath
+    into "${buildDir}/dependencies/main/libs"
+}
+task copyTestDependencies(type: Copy) {
+    from configurations.testCompileClasspath - configurations.compileClasspath
+    into "${buildDir}/dependencies/test/libs"
+}
+
+task copyDependencies {
+  dependsOn(copyMainDependencies)
+  dependsOn(copyTestDependencies)
+}
```

Comment:
this is for C* IDE support.  IntelliJ only knows are dependencies from directories, so to make sure accord works I need to dump dependencies (main/test) and let IntelliJ know about them.